integrations/vello_svg/src/lib.rs

```diff
@@ -175,8 +174,47 @@ fn paint_to_brush(paint: &usvg::Paint, opacity: usvg::Opacity) -> Option<Brush>
             color.blue,
             opacity.to_u8(),
         ))),
-        usvg::Paint::LinearGradient(_) => None,
-        usvg::Paint::RadialGradient(_) => None,
+        usvg::Paint::LinearGradient(gr) => {
+            let stops: Vec<Color> = gr
```

Comment:
I'd suggest converting these to `peniko::ColorStop` so we can also capture the `offset` field. As is, this only works properly if the stops are evenly spaced.